packages/obojobo-chunks-multiple-choice-assessment/viewer-component.js

```diff
@@ -249,7 +249,7 @@ export default class MCAssessment extends React.Component {
 		}
 	}
 
-	componentWillMount() {
+	UNSAFE_componentWillMount() {
```

Comment:
Could this be moved to the constructor or componentDidMount? that's the prefered React Lifecycle